qshift/templates/quarkus-application/manifests/argocd/${{values.component_id}}-db.yaml

```diff
@@ -0,0 +1,41 @@
+{% if values.database == 'quarkus-jdbc-postgresql' %}
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: ${{ values.component_id }}-db
+  namespace: ${{ values.gitopsNamespace }}
+  finalizers:
+    - resources-finalizer.argocd.argoproj.io
+spec:
+  project: default
+  destination:
+    server: https://kubernetes.default.svc
+    namespace: ${{ values.appNamespace }}
+  source:
+    chart: postgresql
+    repoURL: https://charts.bitnami.com/bitnami
+    targetRevision: 11.9.13
+    helm:
+      values: |
+        auth.username: ${{ values.username }}
+        auth.password: ${{ values.password }}
```

Comment:
I think that we should remove password from here. We shouldn't  be providing the password ourselves.
We should let helm use a generated one and pick ints value from the secret.

Reply:
> I think that we should remove password from here. We shouldn't be providing the password ourselves.
> We should let helm use a generated one and pick its value from the secret.

This is something that we will do in a next PR. Current bitnami helm chart dont generate a secret at all ! Primaza will help us here ;-)